### ISSUE TITLE
Depth map always predicts cars to be further than they should

### ISSUE THREAD

**Issue by SohamTamba** [nianticlabs/monodepth2#174]
Hi everyone,

My disparity map always ends up predicting the distance of cars further than they should be, while the rest works fine. The training set involves videos of the 6 monocular views of a car.

Ground Truth
![CAM_BACK](https://user-images.githubusercontent.com/11683616/80851916-88b2ff00-8bf2-11ea-8967-7d2d36378b5a.jpeg)

Disparity Map
![CAM_BACK_disp](https://user-images.githubusercontent.com/11683616/80851925-8f417680-8bf2-11ea-99a5-82fb1b742657.jpeg)

---
I trained the network for 10 epochs using the default hyper-parameters.
There are 89280 training items and 10416 validation items.

Loss at the end:
Training
epoch   9 | batch   1040 | examples/s:  14.4 | loss: 0.06969 | time elapsed: 18h13m54s | time left: 61h33m34s
epoch   9 | batch   3040 | examples/s:  14.2 | loss: 0.08299 | time elapsed: 18h46m00s | time left: 61h01m09s
epoch   9 | batch   5040 | examples/s:  12.6 | loss: 0.06738 | time elapsed: 19h18m02s | time left: 60h28m32s
epoch   9 | batch   7040 | examples/s:  12.3 | loss: 0.06693 | time elapsed: 19h50m18s | time left: 59h56m39s

**Comment by mrharicot** [comment]
Hi,
This is a well known problem in depth models trained from video sequences, please see section 3.2 of the paper.
https://arxiv.org/pdf/1806.01260.pdf
